src/Microsoft.Azure.WebJobs.Host/Config/RetryOptions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host.Config
```

Comment:
Namespace should be Microsoft.Azure.WebJobs.Host. See the other config types.

Reply:
There are 3 different namespaces in the folder, I was not sure which one to use.
```
Microsoft.Azure.WebJobs (JobHostOptions)
Microsoft.Azure.WebJobs.Host (JobHostFunctionTimeoutOptions)
Microsoft.Azure.WebJobs.Host.Config (ConcurrencyOptionsSetup)
```
Changed to Microsoft.Azure.WebJobs.Host